apps/passport-server/src/services/telegramService.ts

```diff
@@ -438,76 +388,6 @@ export class TelegramService {
     });
   }
 
-  private generateProofUrl(telegramUserId: string): string {
```

Comment:
This function is only used in the /start menu, it doesn't need to be a class member so I moved it to the helper file. Also easier to pass in the correct validEventIds as we will see